src/data/news/coverage.yml

```diff
@@ -1,3 +1,27 @@
+-
+  Title: "Decred v1.8.0 - 'The Forkening' feat. Lead Dev Dave Collins"
+  Permalink: "https://www.youtube.com/watch?v=TJnP_t4dEq8"
+  Params:
+    author: "Phoenix Green, Exitus"
+    sortDate: "2023-08-06"
+    icon: "broadcasts.svg"
+    outlet: "Decred YouTube"
+-
+  Title: "Crypto Integrations - Cake Wallet"
+  Permalink: "https://www.youtube.com/watch?v=0KKsD4ZhZn0"
+  Params:
+    author: "Phoenix Green, Tivra"
+    sortDate: "2023-08-02"
+    icon: "broadcasts.svg"
+    outlet: "Decred YouTube"
+-
+  Title: "The Future Is Now: Jake Yocom-Piatt Of Decred.org On How Their Technological Innovation Will Shake Up The Tech Scene"
```

Comment:
It's minor so feel free to ignore, but I always hate seeing caps in URLs. `decred.org` looks way more natural imo.

Reply:
The title was directly copied from the source article. I think it is not uncommon for some websites to capitalize domain names that should really be not capitalized.

Fixed that cap just for our comfort here. In the future we should streamline all titles to `Title Case` or `Sentence case` and fix all nits like this (related to bits of #1101).